ocaml/xapi/message_forwarding.ml

```diff
@@ -988,14 +988,17 @@ functor
         Xapi_pool_helpers.with_pool_operation ~__context
           ~doc:"Pool.enable_tls_verification" ~self ~op:`tls_verification_enable
           (fun () ->
+            debug "Pool.enable_tls_verification start ... (1/3)" ;
             Cert_distrib.exchange_certificates_among_all_members ~__context ;
```

Comment:
If you want to add a fistpoint for testing as @psafont  suggested, I'd have thought here is a good place (just after the exchange)